buildSrc/src/main/groovy/org/elasticsearch/gradle/doc/SnippetsTask.groovy

```diff
@@ -66,6 +66,7 @@ class SnippetsTask extends DefaultTask {
         exclude 'build.gradle'
         // That is where the snippets go, not where they come from!
         exclude 'build'
+        exclude 'build-idea'
```

Comment:
there's also `build-eclipse`. The reason we have these is to make sure builds from the IDE don't interfere with those on the command line.